mod_reforged/hooks/entity/tactical/enemies/necromancer.nut

```diff
@@ -17,6 +17,7 @@
 		this.getSprite("body").Saturation = 0.6;
 		this.m.Skills.add(::new("scripts/skills/actives/raise_undead"));
 		this.m.Skills.add(::new("scripts/skills/actives/possess_undead_skill"));
+		this.m.Skills.add(::new("scripts/skills/perks/perk_rf_soul_link"));
```

Comment:
Should be moved down to the section that is marked with the `//Reforged` comment.